.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,19 +1,45 @@
+Thanks for your interest in Kubecost!  To ensure you get the help you need, please follow these guidelines:
+
+- If you have questions about product functionality/usage or need assistance with product installation/configuration, please email support@kubecost.com, where our support engineering team will be happy to help!
+- If you have a reproducible bug or a feature request, you're in the right place.  Please include the information requested below.
+
+
 ---
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: enhancement
-assignees: ''
+# For reproducible bugs, please provide the following
+
+**Describe the bug**  
+A clear and concise description of what the bug is.
+
+**To Reproduce**  
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. See error
+
+**Expected behavior**  
+A clear and concise description of what you expected to happen.
+
+**Screenshots**  
+If applicable, add screenshots to help explain your problem.
+
+**Collect logs (please complete the following information):**
+- Run `helm ls` and paste the output here:
+- If the pod is stuck in init, run `kubectl logs <kubecost-cost-analyzer pod name> -n kubecost -c cost-analyzer-init` and paste output here:
+- If a page is broken or not loading, open your javascript developer console and paste the ouput here:
+- Run `kubectl logs <kubecost-cost-analyzer pod name> -n kubecost -c cost-analyzer-server` and paste the output here:
+- Run `kubectl logs <kubecost-cost-analyzer pod name> -n kubecost -c cost-model` and paste the output here:
+- Run `kubectl logs <kubecost-cost-analyzer pod name> -n kubecost -c cost-analyzer-frontend` and paste the output here:
```

Comment:
I think this template is for a feature request, not a bug report right? Probably shouldn't have repro and log asks here.